### DISCUSSION TITLE
Is there any way to compensate the rejected frames caused by stitching algorithm

### DISCUSSION THREAD

**Discussion by thilnguy** [OpenStitching/stitching#198] [Q&A]
Hello,
I would like to keep the same ratio of panorama in UI. However, if the stitching algorithm cannot match all images, is there any way/configurations to compensate this to maintain almost the same panorama ratio?
Thanks a lot for your help.

**Comment by lukasalexanderweber** [comment]
need more information. sketch or sample images would be helpful

**Reply by thilnguy** [reply]
![IMG_1582](https://github.com/OpenStitching/stitching/assets/48985195/6250cbb3-5850-4cef-8ae3-8e337fd42451)
Here you are

**Reply by lukasalexanderweber** [reply]
Are your cameras always at the same Position?

**Reply by thilnguy** [reply]
Yes, it is fixed to the ground but it can pan around. Sometimes, the tilt can change a little bit

**Reply by lukasalexanderweber** [reply]
> is there any way/configurations to compensate this to maintain almost the same panorama ratio

no. You would need to implement something that works for your use case. If your cameras would be fixed https://github.com/OpenStitching/stitching/discussions/148 would be for you.

**Reply by thilnguy** [reply]
Thank you for your point.
In my case, I got the warning message: Not all images are included in the final panorama. I tried to reduce the confidence_threshold to 0.1 to make sure all matchs are acceptatble. But it doesnt solve the problem. Do you have any suggestion?
Here is my graph: 
graph matches_graph{
"1" -- "2"[label="Nm=119, Ni=118, C=2.70023"];
"1" -- "3"[label="Nm=8, Ni=8, C=0.769231"];
"4" -- "5"[label="Nm=9, Ni=8, C=0.747664"];
"5" -- "6"[label="Nm=8, Ni=7, C=0.673077"];
"8" -- "9"[label="Nm=51, Ni=47, C=2.01717"];
"8" -- "10"[label="Nm=8, Ni=7, C=0.673077"];
"9" -- "11"[label="Nm=7, Ni=7, C=0.693069"];
"11" -- "12"[label="Nm=40, Ni=35, C=1.75"];
"11" -- "13"[label="Nm=11, Ni=9, C=0.79646"];
"7";
}

**Reply by thilnguy** [reply]
It works well when I reduce the match_conf

**Reply by lukasalexanderweber** [reply]
Thats great!

**Reply by lukasalexanderweber** [reply]
More information: https://github.com/OpenStitching/stitching/discussions/103
